tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/DefaultTaggedMetricRegistry.java

```diff
@@ -14,60 +14,61 @@
  * limitations under the License.
  */
 
-package com.palantir.tritium.metrics;
+package com.palantir.tritium.metrics.registry;
 
 import com.codahale.metrics.Counter;
 import com.codahale.metrics.ExponentiallyDecayingReservoir;
 import com.codahale.metrics.Gauge;
 import com.codahale.metrics.Histogram;
 import com.codahale.metrics.Meter;
 import com.codahale.metrics.Metric;
-import com.codahale.metrics.MetricRegistry;
 import com.codahale.metrics.Timer;
 import java.util.Collections;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.function.Supplier;
 
-/**
- * Similar to {@link MetricRegistry} but allows tagging of {@link Metric}s.
- */
-public final class TaggedMetricRegistry {
+public final class DefaultTaggedMetricRegistry implements TaggedMetricRegistry {
```

Comment:
Do you want to include the service registry info for this in `META-INF/services/com.palantir.tritium.metrics.registry.TaggedMetricRegistry`?

Reply:
since the interface and impl end up in the same jar the service loader should already be able to find the impl, right? can add it for completeness though.